clients/linux/src/account.rs

```diff
@@ -369,16 +413,43 @@ impl Editor {
         cntr.add_named(&info, "folderinfo");
         cntr.add_named(&scroll, "scroll");
 
+        let highlighter = LanguageManager::get_default().unwrap_or_default();
+
+        match Self::language_specs_in_data_dir() {
+            Ok(path) => {
+                let paths = highlighter.get_search_path();
+                let mut str_paths: Vec<&str> = paths.iter().map(|path| path.as_str()).collect();
+                str_paths.push(path.as_str());
+                highlighter.set_search_path(str_paths.as_slice());
+            }
+            Err(err) => match err.target() {
+                LbErrTarget::Dialog => m.send_err_dialog("language specs", err),
+                LbErrTarget::StatusPanel => m.send_err_status_panel(err.msg()),
+            },
+        }
+
         Self {
             info,
             textarea,
-            highlighter: LanguageManager::new(),
+            highlighter,
             change_sig_id: RefCell::new(None),
             cntr,
             messenger: m.clone(),
         }
     }
 
+    fn language_specs_in_data_dir() -> LbResult<String> {
+        let language_specs = format!("{}/language-specs", get_data_dir());
+        let language = format!("{}/custom.lang", language_specs);
+
+        if !Path::new(&language_specs).exists() {
+            std::fs::create_dir(&language_specs).map_err(LbError::fmt_program_err)?;
+            std::fs::write(language, CUSTOM_LANG).map_err(LbError::fmt_program_err)?;
+        }
+
+        Ok(language_specs)
+    }
+
```

Comment:
I don't like seeing file i/o code mixed in with gui code unless it's absolutely necessary. Is this something that can be done when the app is initialized on startup?